docker-compose/cassandra/v3/docker-compose.yaml

```diff
@@ -4,6 +4,8 @@ services:
     ports:
       - "9042:9042"
       - "9160:9160"
+    command: >
+      /bin/sh -c "echo 'authenticator: PasswordAuthenticator' >> /etc/cassandra/cassandra.yaml && docker-entrypoint.sh cassandra -f"
```

Comment:
So this change only enables password authenticator. Is it hardcoded to recognize "cassandra" as username / password?

Reply:
> So this change only enables password authenticator. Is it hardcoded to recognize "cassandra" as username / password?

No, they are  credentials of the default superuser for Cassandra.
See the info in [Cassandra website](https://cassandra.apache.org/doc/stable/cassandra/operating/security.html#authentication)